src/main/resources/application-local.yaml

```diff
@@ -20,3 +20,13 @@ gridsuite:
       base-uri: http://localhost:5022
     report-server:
       base-uri: http://localhost:5028
+
+management:
+  metrics:
+    distribution:
+      percentiles-histogram:
+        http.server.requests: true
+  endpoints:
+    web:
+      exposure:
+        include: prometheus, health, info
```

Comment:
didn't we say it would be in [powsybl-ws-commons/resources/application.yaml](https://github.com/powsybl/powsybl-ws-commons/blob/main/src/main/resources/application.yaml) ?

Reply:
I think that was not really validated. And I'm not convinced we should do it, it means that by default in the build we expose on all our services an endpoint that is really dependent upon a specific tech choice : Prometheus. We could expose our metrics to other TSDB systems. Here it is just in application-local for local start of the application, so for developers, but not sure we should integrate it in the build by default.